app/scripts/lib/setupSentry.js

```diff
@@ -30,6 +30,53 @@ export const SENTRY_BACKGROUND_STATE = {
   AccountTracker: {
     currentBlockGasLimit: true,
   },
+  PermissionLogController: {
+    // double-check if contains PID?
+    permissionHistory: true,
```

Comment:
These entries would have origins, so maybe capturing the types is enough for now. If we want more insights from this, we could update the mask function to mask dynamic collections (arrays/objects), which isn't supported yet

Reply:
Resolved in https://github.com/MetaMask/metamask-extension/pull/20595/commits/b3330828235d61e68ab356e3e859a30d3c44af6f